cmd/dagger/functions.go

```diff
@@ -34,7 +34,7 @@ const (
 
 var funcGroup = &cobra.Group{
 	ID:    "functions",
-	Title: "Function Commands",
+	Title: "Functions",
```

Comment:
You can remove this as it's already integrated in:
- https://github.com/dagger/dagger/pull/7286